config/crd/bases/argoproj.io_applications.yaml

```diff
@@ -28,10 +28,6 @@ spec:
       name: Revision
       priority: 10
       type: string
-    - jsonPath: .spec.project
-      name: Project
-      priority: 10
-      type: string
```

Comment:
Can you run `make generate manifests bundle` so that change is reflected into manifest files?